lib/compat/wordpress-6.0/block-gallery.php

```diff
@@ -14,13 +14,13 @@
  * the editor store and standard flags are not loaded yet at that point
  *
  * @since 12.1.0
- * @todo This should be removed when the minimum required WP version is >= 5.9.
+ * @todo This should be removed when the minimum required WP version is >= 6.0.
  *
  * @return void.
  */
 function gutenberg_check_gallery_block_v2_compatibility() {
 	$use_balance_tags   = (int) get_option( 'use_balanceTags' );
-	$v2_gallery_enabled = boolval( 1 !== $use_balance_tags || is_wp_version_compatible( '5.9' ) ) ? 'true' : 'false';
+	$v2_gallery_enabled = boolval( 1 !== $use_balance_tags || is_wp_version_compatible( '6.0' ) ) ? 'true' : 'false';
```

Comment:
Not sure yet if we want to postpone Gallery v1 removal for 6.1..

Reply:
Let's keep increasing the version number until we get a confirmation from the Mobile team that it can be safely removed. Maybe, we should include the link to the comment from this PR https://github.com/WordPress/gutenberg/pull/41306#issuecomment-1140970131? It looks like usage is dropping below 18%, so 6.1 seems like a good estimate.

Reply:
I think we are okay without a comment as it's something huge and hard to miss 😄 . When we remove the support many many files need to be removed.

Reply:
@fluiddot it would be great if you could make some quick tests about Gallery, even though I didn't remove anything, I still moved some code.. Do you have some bandwidth for it? 😄

Reply:
Hey @ntsekouras 👋 , I don't have too much bandwidth today but I'll try to spare some time to test this as soon as possible, thanks 🙇 !

Reply:
Thank you! I actually merged on the same time, because we made some tests with Gallery as well. If you find anything though, please report.

Reply:
Sorry for the late reply, I've just checked the changes following [these testing instructions](https://github.com/WordPress/gutenberg/pull/41533) and confirm that it works as expected 🎊 .